QuickStart.md

```diff
@@ -0,0 +1,219 @@
+
+# Dependencies
+
+You will need to add the Ride core dependency to your pom:
+```
+    <dependency>
+      <groupId>com.adobe.ride</groupId>
+      <artifactId>ride-core</artifactId>
+      <version>1.0.2</version>
+    </dependency>
+```
+
+# Setup
+
+This quick start will focus on eclipse/maven build workflows, but ant and gradle and any other Java IDE should work just fine too.  To get started, create a new Maven project (sorry, Ride is java-only for now) in the IDE of your choice, giving it a groupId and artifactId you'd like to use (i.e. something like com.mycompany.mywebservice /  webservice-automation-library).  Add the following files to your project (don't worry about the contents, for now just create the place holders:
+
+#### ResourceFiles
+src/main/resources
+	- schemas
+		- SampleService
+			sample\_service\_object\_1.json
+	- configs
+		localhost.properties
+		stage01.properties
+		prod.properties
+		
+#### Java Packages/Classes
+src/main/java
+	- com.mycompany.mywebservice.core
+		- MyWebServiceController.java
+	- com.mycompany.mywebservice.objects
+		- RequestObject.java
+		
+# Resource Code
+
+## Schemas
+
+Open request_object.json and populate it with the schema for your service call payload.  It should look something like this (in most cases, quite a lot more complex):
+```
+{
+	"$schema": "http://json-schema.org/draft-06/schema",
+	"title" : "sample service request object 1",
+	"type": "object",
+	"description": "sub_object",
+	"required": ["name", "type"],
+	"additionalProperties": false,
+	"properties": {
+		"type":  { "enum" : ["type1", "type2", "type3"]},
+		"name": { "type" : "string", "pattern":"^[a-zA-z0-9]{3,30}"},
+		"objectCode": { "type" : "integer", "minimum":1, "maximum":5},
+		"thisProperty": { "type" : "string"}
+	}
+}
+```
+
+
+## Configs
+
+Open the localhost.properties file and add the following code:
+```
+	declaredServices=SampleService
+	isProduction=false
+	
+	#SampleService
+	SampleService.schema=http
+	SampleService.port=80
+	SampleService.endpoint=localhost
+	SampleService.basePath=/sample-service-server/rest
+```
+
+Open the stage.properties file and add the following code:
+
+```
+	declaredServices=SampleService
+	isProduction=false
+	
+	# SampleService
+	SampleService.schema=https
+	SampleService.port=443
+	SampleService.endpoint=www.mycompany.com
+	SampleService.basePath=/sample-service-server/rest 
+```
+	
+Alter the instances of "SampleService" to reflect the actual name of your service, and change your schema, ports, endpoints, and basepaths to the actual ones your service uses (if no basepath, just use /).  Note: whatever you name your service your service name here should also match the name of your folder in the resources schemas folder.
```

Comment:
whatever you name your service your service name here should -- too many words

Reply:
Fixed.